.github/workflows/docker.yml

```diff
@@ -0,0 +1,59 @@
+# This workflow builds, scans, adds set of tags on every push/PR to main branch
+# and pushes the image to docker hub(only): https://hub.docker.com/repository/docker/persistenceone/persistencecore/
+name: Docker CI
+
+on:
```

Comment:
We should be able to run it manually as well.
This will be needed when the `*-fh` tag is pushed, which don't have a release associated with it.

Reply:
We can do this on "fh*" tag release event, rather than a manual workflow. Do you agree?